Sources/LaunchItem.swift

```diff
@@ -28,6 +28,7 @@ public enum LaunchItem: CustomStringConvertible, Equatable {
     @available(iOS 9.0, *)
     case shortcut(item: UIApplicationShortcutItem)
     case userActivity(item: NSUserActivity)
+    case sourceApplication(item: String)
```

Comment:
nit: `case sourceApplication(bundleIdentifier: String)`? (Assuming that the string is indeed the bundle identifier)

Reply:
Yeah, it is.  Went with `item` to match the other cases, but I like that better.